Google.Api.Generator/Generation/ServiceDetails.cs

```diff
@@ -50,7 +50,7 @@ public ServiceDetails(ProtoCatalog catalog, string ns, ServiceDescriptor desc, S
             ProtoPackage = desc.File.Package;
             PackageVersion = ProtoPackage.Split('.').FirstOrDefault(part => ApiVersionPattern.IsMatch(part));
             DocLines = desc.Declaration.DocLines().ToList();
-            SnippetsNamespace = $"{ns}.Snippets";
+            SnippetsNamespace = $"GoogleCSharpSnippets";
```

Comment:
Nit: no need to be interpolated now. Might be worth a comment though to explain how we've arrived at this.

Reply:
Both done, I'll merge on green, and happy to wait till we have more changes here.